tests/core/contracts/test_contract_transact_interface.py

```diff
@@ -9,36 +9,34 @@
 from web3.utils.empty import (
     empty,
 )
-from web3.utils.transactions import (
-    wait_for_transaction_receipt,
-)
+
```

Comment:
Removing this line will fix the lint error

Reply:
@carver thanks, I was having some troubling running tests on my computer so I just pushed and hoped for the best with Travis. Something with parity integration tests. Is there some documentation on how to make that work? Some kind of FileNotFound error.